app/hosts/new/controller.js

```diff
@@ -1,30 +1,20 @@
 import Ember from 'ember';
+import DriverChoices from 'ui/utils/driver-choices';
```

Comment:
I left this in cause i'll be using it in my refactor of drivers. I was just decoupling my changes for the Driver changes but just thought i'd leave this in.